ecms_base/modules/custom/ecms_layout/src/Plugin/Layout/ThreeColumnLayout.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\ecms_layout\Plugin\Layout;
+
+/**
+ * Provides a plugin class for one column layouts.
+ */
+final class ThreeColumnLayout extends LayoutBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getColumnWidths(): array {
+    return [];
```

Comment:
Should there be column options here?

Reply:
No, with three column options we do no not want to give them options for col size. They will all be 33% - 33% - 33%. This applies to 4 col layouts as well.